composer.json

```
@@ -21,13 +21,13 @@
     "require": {
         "php": "^7.1",
         "sonata-project/cache": "^2.0.0",
-        "symfony/config": "^3.4 || ^4.0",
-        "symfony/dependency-injection": "^3.4 || ^4.0",
-        "symfony/filesystem": "^3.4 || ^4.0",
-        "symfony/http-foundation": "^3.4.35 || ^4.0",
-        "symfony/http-kernel": "^3.4 || ^4.0",
+        "symfony/config": "^3.4 || ^4.0 || ^5.0",
```

Please change this to:
```suggestion
        "symfony/config": "^4.3 || ^5.0",
```

We won't support 3 major versions of symfony